internal/kustomize/kustomize.go

```diff
@@ -80,3 +75,24 @@ func Render(
 	// Glue the manifests together
 	return manifests.CombineYAML(yamlManifests), nil
 }
+
+func BuildKustomizeOptions(
+	enableHelm bool,
+	loadRestrictor string,
+) *argoappv1.KustomizeOptions {
+	buildOptions := ""
+
+	if enableHelm {
+		buildOptions += "--enable-helm "
+	}
+
+	if loadRestrictor != "" {
+		buildOptions += fmt.Sprintf("--load-restrictor %s", loadRestrictor)
+	} else {
+		buildOptions += "--load-restrictor LoadRestrictionsRootOnly"
+	}
```

Comment:
If this is the default behavior, we could just not include the flag in this case.

Reply:
It's also fine if you want to leave it. Dealer's choice.